cmake/projects/drm/hunter.cmake

```diff
@@ -22,7 +23,17 @@ hunter_add_version(
 )
 
 hunter_configuration_types(drm CONFIGURATION_TYPES Release)
-hunter_pick_scheme(DEFAULT drm)
+hunter_pick_scheme(DEFAULT url_sha1_autotools)
+list(APPEND drm_dependencies
```

Comment:
> This file can be processed several times

Oh, ok, I thought you removed include guards because this file would only be processed once. I guess I misinterpreted that change :-)